chess/templates/chess/signup.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'base.html' %}
+{% load static %}
+{% block content %}
+<div class="center-container">
+    <form action="" method="post">
+        {% csrf_token %}
+        {% for field in form %}
+            <div>
+                {{field.label}}
+                {{field}}
+                {{field.error}}
+            </div>
```

Comment:
We need to reformat these forms so they are properly displayed in a neat order and add better looking CSS.